glossary.yml

```diff
@@ -907,8 +922,9 @@
   en:
     term: "catch (an exception)"
     def: >
-      To accept responsibility for handling an error or other unexpected event. R
-      prefers "handling a condition" to "catching an exception".
+      To accept responsibility for handling an error or other unexpected event. [R](#r_language)
+      prefers "handling a condition" to "catching an exception". [Python](#python), on the other hand,
```

Comment:
Unless I am mistaken, this is going to be flagged by the linter because `handle_condition` is in both the `ref` list, and linked internally. I had to remove a bunch of links I added in for this reason.

(It is possible I accepted some such links already, without realising they were removed from my pull request to satisfy the linter.)

Reply:
I think you are correct! Forgive the intrusion.